File: CS/SpreadsheetControl_WPF_API/BusinessObjects.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows;
using System.Windows.Controls;
using System.Windows.Data;
using System.Windows.Documents;
using System.Windows.Input;
using System.Windows.Media;
using System.Windows.Media.Imaging;
using System.Windows.Navigation;
using System.Windows.Shapes;
using DevExpress.Xpf.NavBar;
using DevExpress.Spreadsheet;


namespace SpreadsheetControl_WPF_API
{
    public class Group
    {
        public string Header { get; set; }
        public List<SpreadsheetExample> Items { get; set; }

        public Group(string name)
        {
            Header = name;
            Items = new List<SpreadsheetExample>();
        }        
    }

    public class SpreadsheetExample
    {
        public string Header { get; set; }
        public SpreadsheetExample(string name, Action<IWorkbook> action)
        {
            Header = name;
            Action = action;
        }
        public Action<IWorkbook> Action { get; private set; }
    }
}

File: VB/SpreadsheetControl_WPF_API/BusinessObjects.vb
Imports System
Imports System.Collections.Generic
Imports System.Windows.Controls
Imports DevExpress.Spreadsheet

Namespace SpreadsheetControl_WPF_API

    Public Class Group

        Public Property Header As String

        Public Property Items As List(Of SpreadsheetExample)

        Public Sub New(ByVal name As String)
            Header = name
            Items = New List(Of SpreadsheetExample)()
        End Sub
    End Class

    Public Class SpreadsheetExample

        Private _Action As Action(Of DevExpress.Spreadsheet.IWorkbook)

        Public Property Header As String

        Public Sub New(ByVal name As String, ByVal action As Action(Of IWorkbook))
            Header = name
            Me.Action = action
        End Sub

        Public Property Action As Action(Of IWorkbook)
            Get
                Return _Action
            End Get

            Private Set(ByVal value As Action(Of IWorkbook))
                _Action = value
            End Set
        End Property
    End Class
End Namespace
